index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <main id="main">
+      <section class="heading">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+      </section>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+        />
+        <caption id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </caption>
```

Comment:
- [ ] `required`
- [ ]  Amazing Job so far 👏 👏
- [ ]  I suggest you add emoji's to your comments so that the student understands the tone you are trying to communicate
- [ ] I also strongly suggest that in order to sound polite you need to be using worlds like `please` `kindly` and `i suggest` and many more other worlds
- [ ] Also Please explain `why` you are asking for that that change so the student can learn something as well and don't feel the change was unfair
 Please make amends ✌️

Reply:
Thanks for the feedback, and I'll do so.